templates/partials/answers/checkbox.html

```diff
@@ -1,10 +1,12 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
+{% set number_of_checkboxes = answer.options|length %}
```

Comment:
```suggestion
{% set number_of_checkboxes = answer.options | length %}
```

Reply:
Any reason why? Don't tend to have gaps in the jinja docs I've seen

Reply:
Personally, I think it is more readable and the majority of places in runner are separated by a space when using `|` operator. Would be nice is it was fully consistent.